components/base/form/VFile.vue

```diff
@@ -60,6 +76,16 @@ const field = inject<any>("field");
 const imageIsTooSmall = ref(false);
 const aspectRatioIsWrong = ref(false);
 
+const imageSizeIsTooBig = ref(false);
+const imageSizeIsTooSmall = ref(false);
+
+const videoSizeIsTooSmall = ref(false);
+const videoSizeIsTooBig = ref(false);
+const videoDurationIsTooShort = ref(false);
+const videoDurationIsTooLong = ref(false);
+const videoResolutionIsTooLow = ref(false);
+const videoResolutionIsTooHigh = ref(false);
```

Comment:
Un boolean doit toujours commencer par has ou is

Reply:
Je veux bien qu'on rajoute une nouvelle règle mais soit le fichier sera hybride, soit faut renommer tous les boolean y compris dans les props et donc ça va casser l'intégration du VFile. (Notament pour les props "button" et "multiple".

Reply:
Il n'y a pas de régle dans eslint pour la gérer.